### PR TITLE
gopass: Add passAlias parameter to derivation

### PR DESCRIPTION
**Pull request by rvolosatovs**
###### Motivation for this change
Add a `passAlias` to `gopass` derivation, similarly to how there's `viAlias` parameter supported by vim/neovim derivations.
Apart from convenience this is also necessary for e.g. making `docker-credential-helpers` work without patching the latter - the package uses `pass` executable from `PATH` https://github.com/docker/docker-credential-helpers/blob/f78081d1f7fef6ad74ad6b79368de6348386e591/pass/pass_linux.go#L67. By setting `passAlias` to `true` (e.g. via overlays like so: https://github.com/rvolosatovs/infrastructure/blob/94971f34db9df79b6666e626fa52840eb1fee840/nixpkgs/overlays.nix#L315-L317), secure Docker authentication via `gopass` "just works"™

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rvolosatovs** [comment]
cc @andir

**Comment by rvolosatovs** [comment]
@GrahamcOfBorg build gopass

**Comment by andir** [comment]
Looks good to me. Could you please make sure the reasoning you provided in the PR description is also in the commit?

**Comment by rvolosatovs** [comment]
@andir done, rebased on latest `master` as well.

**Comment by rvolosatovs** [comment]
@GrahamcOfBorg build gopass

File: pkgs/tools/security/gopass/default.nix [hunk 1]
```diff
@@ -1,4 +1,11 @@
-{ stdenv, buildGoModule, fetchFromGitHub, git, gnupg, xclip, wl-clipboard, installShellFiles, makeWrapper }:
+{ stdenv, makeWrapper
+, buildGoPackage, fetchFromGitHub, installShellFiles
```
Comment (andir) [inline pkgs/tools/security/gopass/default.nix:2]: You changed this here to `buildGoPackage` but further down you are using `buildGoModule`

Comment (rvolosatovs) [reply pkgs/tools/security/gopass/default.nix]: Oops, yes, that seems like a rebase artifact - the package was using `buildGoPackage` before

Comment (rvolosatovs) [reply pkgs/tools/security/gopass/default.nix]: Fixed it (and built locally as well this time :P)

**Comment by rvolosatovs** [comment]
@GrahamcOfBorg build gopass

**Pull request merged**